Improve networking panels to work with read write deployment mode

pracucci opened the pull request:
#### What this PR does
Follow up the work done in https://github.com/grafana/mimir/pull/3497 and https://github.com/grafana/mimir/pull/3504, it this PR I'm addressing the "networking" dashboards. In details:

- Added "Summary" row both to "Writes networking" and "Reads networking" dashboards
- Changed "Inflight requests" and "TCP connections" queries to not match read-write deployment mode when building the panels for a specific component (e.g. "distributor"). This is the same pattern used for "Writes/Reads resources" dashboards too

**Example microservices deployment:**
![Screenshot 2022-11-28 at 10 48 10](https://user-images.githubusercontent.com/1701904/204246594-0093d9d9-898b-455c-982e-bbe8d3f9c1a9.png)


**Example read-write deployment:**
![Screenshot 2022-11-28 at 10 48 04](https://user-images.githubusercontent.com/1701904/204246584-73d4b234-84e4-4117-b48f-53b6dcf756f3.png)


#### Which issue(s) this PR fixes or relates to

Part of #3361

#### Checklist

- [ ] Tests updated
- [ ] Documentation added
- [x] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

dimitarvdimitrov reviewed (approved):
jsonnet changes make sense to me, but the Summary row is not in the screenshots in the PR description. I tried generating them locally, but the screenshot taker seems to fail right after it reaches "Taking screenshot of mimir-overview-resources"

edit: `make mixin-screenshots` fails with 

```
Taking screenshot of mimir-overview-resources
node:internal/errors:844
  const err = new Error(message);
              ^

Error: Command failed: pngquant --force --ext '.png' --skip-if-larger --speed 1 --strip --quality 100 /output/overview-resources/mimir-overview-resources.png

    at ChildProcess.exithandler (node:child_process:398:12)
    at ChildProcess.emit (node:events:527:28)
    at maybeClose (node:internal/child_process:1090:16)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:302:5) {
  code: 99,
  killed: false,
  signal: null,
  cmd: "pngquant --force --ext '.png' --skip-if-larger --speed 1 --strip --quality 100 /output/overview-resources/mimir-overview-resources.png",
  stdout: '',
  stderr: ''
}

Node.js v17.9.1
```

pracucci commented:
> jsonnet changes make sense to me, but the Summary row is not in the screenshots in the PR description. I tried generating them locally, but the screenshot taker seems to fail right after it reaches "Taking screenshot of mimir-overview-resources"

The original commit had a bug. The "Summary" row was called "Query-frontend". Then I fixed it, but forgot to update the example screenshots in the PR description. I just updated them.

pracucci commented:
> I tried generating them locally, but the screenshot taker seems to fail right after it reaches "Taking screenshot of mimir-overview-resources"

I just tried to run `make mixin-screenshots` and it succeeded. Please reach out to me on Slack, so we can better investigate it.
